### DISCUSSION TITLE
What's the best way to debug latency issues using cmp ?

### DISCUSSION THREAD

**Discussion by MarcWeber** [hrsh7th/nvim-cmp#1822] [Q&A]
I have a setup with cmp-buffer, intellephense, my own code. And at some point lag turns unbearable feels like 500ms.

So suggestion is like:

1) document how to debug speed issues
What I'd like to see is "echo cmp.last_timings" saying {a: gather 80ms, .. sort: X ms} to find the problem fast

2) document how to enable / disable configurations.
Eg .setup({..}) is used |But what your really want is g:cmp_config.soucres['cmp'].enable = 0

**Comment by Shougo** [comment]
You should create the minimal vimrc file.
We cannot test it.
